README.md

```diff
@@ -12,6 +12,11 @@ To monitor your source directory for changes, and recompile files if they change
 python build.py & python -m SimpleHTTPServer && fg
 ```
 
+For Python 3.6.0 : 
+```
+python build.py & python -m http.server && fg
```

Comment:
This should work for all versions > 3.3

Its better to mention that, 3.6 was released recently, but this should work with relatively newer python 3 versions